lib/receipts/base.rb

```diff
@@ -36,23 +36,21 @@ def setup_fonts(custom_font = nil)
     end
 
     def load_image(logo)
-      if logo.is_a? String
-        logo.start_with?("http") ? URI.parse(logo).open : File.open(logo)
-      else
-        logo
-      end
+      is_url?(logo) ? URI.parse(logo).open : logo
     end
 
     def header(company: {}, height: 16)
       logo = company[:logo]
 
-      if logo.nil?
-        text company.fetch(:name), align: :right, style: :bold, size: 16, color: "4b5563"
-      else
+      if logo_is_image?(logo)
         image load_image(logo), height: height, position: :right
+        move_up height
+      elsif logo.nil? || logo != false
```

Comment:
Could probably even just do

```suggestion
      elsif logo.nil? || logo
```